guides/common/attributes-base.adoc

```diff
@@ -29,6 +29,26 @@
 :RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
 :RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
 
+//Added by Marie
+:SatelliteSub: Red Hat Satellite Infrastructure Subscription
+:RepoRHEL7Server: rhel-7-server-satellite
+:RepoRHEL8Server: rhel-8-for-x86_64-baseos-rpms 
+// We would probably need to mention also "rhel-8-for-x86_64-appstream-rpms"
+:RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
+:RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
+// :RepoRHEL7ServerOptional: any equivalent for Sat7 for RHEL 8? - "codeready-builder-for-rhel-8-x86_64-rpms" ?
+:RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
+// :RepoRHEL8ServerAnsible: ansible-2.8-for-rhel-8-x86_64-rpms ???
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-capsule-7.0-rpms
+:RepoRHEL8ServerSatelliteServerProductVersion: satellite-7.0-for-rhel-8-x86_64-rpms
+:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-server-7-satellite-6.10-rpms
+:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-7.0-rpms
+:RepoRHEL8ServerSatelliteCapsuleProductVersion: satellite-capsule-7.0-for-rhel-8-x86_64-rpms
+:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-7.0-rpms
+:RepoRHEL8ServerSatelliteMaintenanceProductVersion: satellite-maintenance-7.0-for-rhel-8-x86_64-rpms
+// Satellite-Tools will become Satellite-Client ??? - I haven't found this repo mentioned anywhere except for this file. So it may not be needed.
+// Repos relevant for Puppet - The Puppet repos were not mentioned in 6.10 docs. 
+
```

Comment:
We have a file attributes-satellite which is included when BUILD=satellite. I wonder if it should all have been there instead of the attributes-base.adoc.

I am assuming that ll of these attributes are only used in "if satellite" statements of course, otherwise they would be rendered directly into text (not expanded).

Reply:
Hi @lzap , yes, it sounds logical. TBH, I'm no sure why it was set like this. I'd need to take a closer look, which may take some time.

Reply:
I totally agree with lzap, I was about to suggest the same.

Reply:
Hi @lzap , so the attributes-base.adoc file says that these attributes are used in both Satellite and Katello docs. I'm not sure  which problems it could cause if we just move them under attributes-satellite.adoc (and maybe we would need to copy also under attributes-katello.adoc). Would it be an issue to keep them in the attributes-base.adoc file? Thank you!

Reply:
AFAIK there are some common ones. I think these are:
```
:RepoRHEL7Server: rhel-7-server-rpms
:RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
:RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
:RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
```

Those make are needed to install Katello on upstream using RHEL 7.

However, the `RepoRHEL7ServerSatellite*` ones are only used by Satellite. So I'd move all those to the Satellite guide. If any upstream source uses it, that's a bug. `SatelliteSub` is another that I suspect is only needed in the Satellite case.

Does that help?

Reply:
agree, yeah, there are repos you need to deploy *any* variant on RHEL which should go to common attributes, and then there are satellite specific ones, which should go to satellite attributes.